lib/gyro/infos.rb

```diff
@@ -0,0 +1,29 @@
+# Gyro Infos
+#
+module Gyro	
+	module Infos
+		TEMPLATE_FILE = "./lib/templates"
+		def self.listTemplates
+			entries = Dir[Infos::TEMPLATE_FILE+"/*"] 
+				entries.each do |entry|
+			  	if File.directory?(entry)
+			  		puts File.basename(entry)
+		  		end
+  			end
+		end
+
+		def self.showInfos(template)
+			entries = [template+"README.md",
```

Comment:
Same idea: using `aPathname + aString` automatically adds the slashes between components (that's one of the nice things that makes working with Pathname great when dealing with paths in 
ruby) 

So 
```ruby
readme = Gyro::Liquid::Generator.templates_dir + templateName + 'README.md'
Gyro::Error.exit_with_error("No documentation found for this template") unless readme.exists?
puts readme.read # or something like this I'm not sure of the method name look at Pathname doc 
```